src/main/java/dev/morling/onebrc/CalculateAverage_davecom.java

```diff
@@ -118,14 +120,20 @@ public static void outputResults() {
         // fast string concatenation starting with { and ending with } with a comma between each (no newlines)
         StringBuilder sb = new StringBuilder();
         sb.append('{');
-        var sortedNames = mins.keySet().stream().sorted().toArray(String[]::new);
+        // var sortedNames = mins.keySet().stream().sorted().toArray(String[]::new);
+
         DecimalFormat df = new DecimalFormat("0.0");
```

Comment:
Note that on my French locale, numbers are showing with a comma instead of dot (e.g. -7,5). You may want to use Locale.setDefault() before.
Thanks for your code, I had forgotten about IntSummaryStatistics class. That simplifies my code.

Reply:
Good point and glad you found it helpful!